ansible/setup.yml

```diff
@@ -70,3 +72,26 @@
         - python-pip
       tags:
         - dev-tools
+
+    - name: Download libsodium
+      unarchive:
+        src: https://github.com/jedisct1/libsodium/releases/download/{{ libsodium_version }}/libsodium-{{ libsodium_version }}.tar.gz
+        dest: /usr/local/src/
+        remote_src: yes
+      args:
+        creates: /usr/local/src/libsodium-{{ libsodium_version }}.tar.gz
```

Comment:
I am surprised unarchive creates a .tar.gz

Reply:
From the documentation at http://docs.ansible.com/ansible/latest/unarchive_module.html this is still unclear to me

Reply:
This was a bug/obsolete configuration fixed in c14dfe6, the module is already idempotent (checks if dest: dir exists).

Reply:
good catch btw